### PR TITLE
Introduce HII feature to timberland_1.0_final

### PR DESCRIPTION
**Pull request by trevor-cockrell**
Needed to rebase https://github.com/timberland-sig/edk2/pull/20 to signoff commits for merge - new PR needed since branches have technically diverged.

@mwilck, @johnmeneghini, refreshed approval requested.

### CODE REVIEW

**Review by mwilck — APPROVED** [review]
Verified that the changes are identical to #20, which I've already approved.

**Comment by johnmeneghini** [comment]
> Verified that the changes are identical to #20, which I've already approved.

@trevor-cockrell  is this PR really identical to #20 ?   Why did you create a new pull request?

**Comment by mwilck** [comment]
@johnmeneghini I think it's just about commit meta data. The code is the same, I verified that using `git range-diff`.

**Comment by Douglas-Farley** [comment]
> @johnmeneghini I think it's just about commit meta data. The code is the same, I verified that using `git range-diff`.

Correct, just had to clean up the verified signatures and some other meta-data needed for upstream;  We didn't have the call this week to raise it there and merge it by votes.

File: NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#string STR_NVMEOF_LOCAL_MASK                     #language en-US "  Host Subnet Mask"
+#string STR_NVMEOF_LOCAL_GATEWAY                  #language en-US "  Gateway"
+#string STR_NVMEOF_IP_ADDRESS_HELP                #language en-US "Enter IP address in dotted-decimal notation."
+#string STR_NVMEOF_ENABLE_DHCP_ON_SUBSYS          #language en-US "Subsystem info from DHCP"
+#string STR_NVMEOF_SUBSYS_NQN                     #language en-US "  NVM Subsystem NQN"
+#string STR_NVMEOF_SUBSYS_NQN_HELP                #language en-US "The target subsystem's NVMe Qualified Name."
+#string STR_NVMEOF_SUBSYS_ADDRESS                 #language en-US "  NVM Subsystem Address"
+#string STR_NVMEOF_SUBSYS_ADDRESS_HELP            #language en-US "Enter NVM Subsystem address in IPv4,IPv6 or URL format. You need to configure DNS server address in advance if input a URL string."
+#string STR_NVMEOF_SUBSYS_PORT                    #language en-US "  NVM Subsystem Port"
+#string STR_NVMEOF_SUBSYS_NID                     #language en-US "NVM Subsystem NID"
```
Comment (johnmeneghini) [inline NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni:65]: What's a Subsystem NID?  There's not such thing. It's called a Namespace Identifier.  Just simply call it a "Namespace Identifier (NID)"  Drop the "NVM Subsystem"

**Review by johnmeneghini — APPROVED** [review]
One little nit.  Please clarify or change this and then merge this.  Thanks!!

File: NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni (hunk 1 above)
Comment (Douglas-Farley) [reply NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni:65]: How about:
```suggestion
#string STR_NVMEOF_SUBSYS_NID                     #language en-US "NVM Namespace Identifier (NID)"
```

Comment (trevor-cockrell) [reply NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni:65]: Tracking this item here:
https://github.com/timberland-sig/edk2/issues/26

**Comment by trevor-cockrell** [comment]
Based on the call from today (9/28): In an effort to keep things moving, merging this branch for now. I've opened issue #26 for tracking the mentioned NID string issue.

**Pull request merged**